src/util/audioHelpers.ts

```diff
@@ -44,9 +29,9 @@ export const audioApiTypeToFormType = (
 
   return {
     ...audio,
-    title: plainTextToEditorValue(title),
-    manuscript: plainTextToEditorValue(manuscript),
-    tags: Array.from(new Set(tags)),
+    title: plainTextToEditorValue(audio?.title?.title ?? ''),
+    manuscript: plainTextToEditorValue(audio?.manuscript?.manuscript ?? ''),
+    tags: audio?.tags.tags ?? [],
```

Comment:
Dette er vel ikke nødvendigvis likt det som var her tidligere. Et `Set` er unikt i at det kun inneholder unike verdier. Vet ikke om det er relevant i denne situasjonen, men viktig å huske på.

Reply:
Godt poeng, hadde det i bakhodet når jeg skrev om. Jeg kikka alle andre steder vi gjør dette og der bruker vi aldri `Set`, vi tar tags direkte slik som det er skrevet nå. Så tror jeg holder meg til det for consistency sin skyld.